examples/18_error_type_passthrough.rs

```diff
@@ -94,38 +101,45 @@ async fn main() -> Result<(), miette::Report> {
         .handle_shutdown_requests(Duration::from_millis(500))
         .await;
 
+    let mut sum = String::new();
```

Comment:
After reading a bit online, it seems that multiple messages are preferred over one multi-line message. Multi-line messages seem quite discouraged.